packages/giflib/giflib.1.0.3/opam

```diff
@@ -0,0 +1,49 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Implementation of GIF image format in OCaml"
+description: "Implementation of GIF image format in OCaml."
+maintainer: [
+  "Michael Dales <michael@digitalflapjack.com>"
+  "Shreya Pawaskar <shreya.pawaskar.main@gmail.com>"
+]
+authors: [
+  "Jan Dudek <jd@jandudek.com>"
+  "Michael Dales <michael@digitalflapjack.com"
+  "Shreya Pawaskar <shreya.pawaskar.main@gmail.com>"
+]
+license: "MIT"
+tags: ["image" "gif" "bitmap"]
+homepage: "https://github.com/claudiusFX/ocaml-gif"
+doc: "https://github.com/claudiusFX/ocaml-gif"
+bug-reports: "https://github.com/claudiusFX/ocaml-gif/issues"
+depends: [
+  "ocaml" {>= "5.1"}
+  "dune" {>= "3.14"}
+  "ounit2"
+  "zarith"
+  "odoc"
+  "ocamlformat" {>= "0.27.0"}
```

Comment:
This does not seem to be needed except for documentation builds and for dev flow:
```suggestion
  "odoc" {with-doc}
  "ocamlformat" {>= "0.27.0" & with-dev-setup}
```